src/Pages/TheTeam/TheTeam.js

```diff
@@ -17,7 +17,7 @@ import Evan from './teamPics/2018_2019/evanUgarte.png'
 // import Surabhi from './teamPics/2019-2020/serbi.png'
 // import Nanar from './teamPics/2019-2020/nanar.png'
 
-export default class TheTeam extends React.Component {
+export default class Example extends React.Component {
```

Comment:
fetch from origin and rebase your branch again. i'm not sure why this is changed

Reply:
Is it still ```Example```